From 9240ea30ee29d2a499bcf3eed9e12dd9921f39f8 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Mon, 3 Jun 2019 23:12:26 +0000
Subject: [PATCH] OS-7769 Want Intel PCH temperature sensor (missing makefile)
 Reviewed by: Patrick Mooney <pmooney@pfmooney.com> Approved by: Patrick
 Mooney <pmooney@pfmooney.com>

---
 .../fm/topo/modules/i86pc/chipset/Makefile    | 29 +++++++++++++++++++
 1 file changed, 29 insertions(+)
 create mode 100644 usr/src/lib/fm/topo/modules/i86pc/chipset/Makefile

diff --git a/usr/src/lib/fm/topo/modules/i86pc/chipset/Makefile b/usr/src/lib/fm/topo/modules/i86pc/chipset/Makefile
new file mode 100644
index 0000000000..cd8c8761f7
--- /dev/null
+++ b/usr/src/lib/fm/topo/modules/i86pc/chipset/Makefile
@@ -0,0 +1,29 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Joyent, Inc.
+#
+
+MODULE = chipset
+ARCH = i86pc
+CLASS = arch
+SHAREDDIR = ../../common/shared/
+
+MODULESRCS = chipset.c topo_sensor.c
+
+include ../../Makefile.plugin
+
+CPPFLAGS += -I$(SHAREDDIR)
+
+%.o: $(SHAREDDIR)/%.c
+	$(COMPILE.c) -o $@ $<
+	$(CTFCONVERT_O)
-- 
2.21.0

